apps/base-docs/docs/tools/account-abstraction.md

```diff
@@ -39,6 +39,12 @@ hide_table_of_contents: true
 
 ---
 
+## Openfort
+
+[Openfort](https://openfort.xyz) is an infrastructure provider designed to simplify the development of games and gamified experiences across their suite of API endpoints. The platform vertically integrates the AA stack, so game developers can focus on game development without worrying about private key management, the account model or the onchain interactions with paymasters and bundlers.
```

Comment:
@joalavedra 

Thanks for the PR! Could you please clarify in the description how Openfort specifically supports Base?

Reply:
WE're deployed in BASE and working with the team as well to offer BASE bundlers and paymasters in the stack